tests/mbed/enc/CMakeLists.txt

```diff
@@ -47,6 +47,30 @@ function(add_mbed_test_enclave NAME)
     ${PROJECT_SOURCE_DIR}/3rdparty/mbedtls/mbedtls/tests
     ${CMAKE_CURRENT_BINARY_DIR}
     ${CMAKE_BINARY_DIR}/3rdparty/mbedtls/mbedtls-wrap-prefix/src/mbedtls-wrap/include/)
+  
+  maybe_build_using_clangw(mbedtest_suite_${NAME})
```

Comment:
The alignment looks off

Reply:
It looks fine with multiple editors and also in GitHub.

Reply:
Maybe you're using tabs? It looks off to me too.

Reply:
Modified to 2 spaces.

Reply:
I manually edited the file to revert to 2 spaces and then ran format-code. But git diff didn't reveal the change. Will look at this again.

Reply:
It's fine, it's just hard to tell here. Thanks Anita, sorry about that.